src/Microsoft.Diagnostics.Runtime/src/Utilities/COMInterop/ComCallableIUnknown.cs

```diff
@@ -115,6 +115,14 @@ private int ReleaseImpl(IntPtr self)
                 // Only free memory the first time we reach here.
                 if (_handle.IsAllocated)
                 {
+                    try
+                    {
+                        Destroy();
+                    }
+                    catch
```

Comment:
Should we have a blanket catch here?  I would think we'd rather not have anyone who implemented `Destory` throw an exception...they could wrap in try/catch.

If the intent is to ensure cleanup, maybe a `try { Destroy(); } finally { ...cleanup code below... }` pattern instead.  We'd end up releasing the memory correctly and then propagate the exception back out.

Reply:
I switch it to a try/finally.